packages/hooks/src/useConfig.ts

```diff
@@ -1,8 +1,28 @@
-import { useContext } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { LiteflowContext } from './context'
 import { Config } from './graphql'
 
-export default function useConfig(): Promise<Config> {
-  const { config } = useContext(LiteflowContext)
-  return config
+// This type can be reused for any other read-only hook
+type Result<T = any> = {
```

Comment:
It should be exported, isn't it?
```suggestion
export type Result<T = any> = {
```

Reply:
I added an intermediary type `ConfigResult` to not export a type with generics. I don't think it's super useful to export but I'm not against it